recipes/avahi-system/all/conanfile.py

```diff
@@ -0,0 +1,44 @@
+from conans import ConanFile, tools
+from conans.errors import ConanInvalidConfiguration
+
+
+class SysConfigAvahiSystemConan(ConanFile):
+    name = "avahi-system"
```

Comment:
I the version system gives it meaning 🤔 but not my area of expertise

Reply:
Hopefully @uilianries can advise. I thought `avahi` maybe should be reserved for a potential recipe building from [lathiat/avahi](https://github.com/lathiat/avahi) source, but maybe distinguishing via `version="system"` is enough...

Reply:
We can have both in the same folder... it's the one recipe per version layout.

The `recipes/avahi/system` can be this and in the future someone can add `recipes/avahi/v1.1.1` and everything works AFAIK

Reply:
Good idea, resolved by cd992f4fdf5dc6c90f96f1b5a6802d515a8dfab4